config/swarm-testing.json

```diff
@@ -1,9 +1,13 @@
 {
   "seedNodeList": [
     {
-      "ip": "localhost",
+      "ip": "127.0.0.1",
       "port": "22129"
     }
+    // {
+    //   "ip": "167.114.135.147",
+    //   "port": "22129"
+    // }
```

Comment:
I don't think this will parse fine

Reply:
Looks like I commited more than I intended...